### PR TITLE
Create gravy page, static query for data, add link, update stackedIma…

### PR DESCRIPTION
**Pull request by johnnycrisp**
Add gravy custom type in Prismic
Add gravy page (mimic studio page)
Update query
Update stacked images to fix conditional build error with captions

### CODE REVIEW

File: src/pages/gravy.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+import CloseButton from "../components/atoms/CloseButton";
+import Gravy from "../components/organisms/Gravy";
+import useInitialGlobalNavSplit from "../hooks/useInitialGlobalNavSplit";
+
+const Container = styled.div`
+  margin-top: 48px;
+`;
+
+const GravyPage = (props) => {
+  useInitialGlobalNavSplit(props.globalNav, 'gravy', 1);
```
[the remark this answers is not in the document]
Comment (johnnycrisp) [reply src/pages/gravy.js]: Got it - done!

File: src/components/organisms/Gravy.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+                      }
+                  }
+              }
+          }
+      }
+  `);
+  return (
+    <Holder>
+      <Inner className="p-large">
+      </Inner>
```
Comment (willmcl) [inline src/components/organisms/Gravy.js:61]: Don't need this Inner

File: src/pages/gravy.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import styled from "styled-components";
+
+import CloseButton from "../components/atoms/CloseButton";
+import Gravy from "../components/organisms/Gravy";
+import useInitialGlobalNavSplit from "../hooks/useInitialGlobalNavSplit";
+
+const Container = styled.div`
+  margin-top: 48px;
```
Comment (willmcl) [inline src/pages/gravy.js:9]: Don't need this. It is adding a 48px header to the page. Can you check it against the live site to make sure it matches?

File: src/components/organisms/Gravy.js [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+function Gravy(props) {
+  const [currentSlide, setCurrentSlide] = useState(0);
+  const data = useStaticQuery(graphql`
+      query GravyQuery {
+          prismicGravy {
+              data {
+                  gallery {
+                    caption {
+                        text
```
Comment (willmcl) [inline src/components/organisms/Gravy.js:42]: Can you fill in the captions on Prismic? We are missing them on the deploy preview.

File: src/pages/gravy.js (hunk 3 above)
Comment (johnnycrisp) [reply src/pages/gravy.js:9]: Removed. Can't see a header on the live site.

File: src/components/organisms/Gravy.js (hunk 2 above)
Comment (johnnycrisp) [reply src/components/organisms/Gravy.js]: Gone

File: src/components/organisms/Gravy.js (hunk 4 above)
Comment (johnnycrisp) [reply src/components/organisms/Gravy.js:42]: There is only a caption on the fist image on the live site.

**Pull request merged**